public/index.ejs

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en-US">
+
+<head>
+  <meta charset="utf-8" />
+  <title>Cluster Application Migration Tool</title>
+  <meta id="appName" name="application-name" content="Mig UI" />
+  <!-- <link rel="icon" type="image/png" href="/assets/favicon.ico"> -->
+  <!-- <link rel="stylesheet" href="../node_modules/@patternfly/react-core/dist/styles/base.css" /> -->
```

Comment:
@ibolton336 I think these are what I need help with. I think I need to copy this file out of the node_modules and expose it from the file server so we load it prior to the app bundle. I'm not sure how to do that in the webpack production build.